<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReservationForm</class>
 <widget class="QWidget" name="ReservationForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SkiRent - reservation</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Equipment:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="equipmentTypeCombo">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Condition:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="equipmentConditionCombo"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Amount:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="amountSpin"/>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Rent from:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="rentFromEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Rent to:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="rentToEdit"/>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
